### PR TITLE
Try new xdebug filter functionality

### PR DESCRIPTION
**Pull request by greg0ire**
## Subject

See https://thephp.cc/news/2019/01/faster-code-coverage

### CODE REVIEW

**Comment by greg0ire** [comment]
I'm probably not using it right because the timing is the same on my machine :(

**Comment by greg0ire** [comment]
Or maybe that's because we don't use a lot of external code in our test suite…

File: xdebug-filter.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+if (!function_exists('xdebug_set_filter')) {
+    return;
+}
+
+xdebug_set_filter(
+    \XDEBUG_FILTER_CODE_COVERAGE,
+    \XDEBUG_PATH_WHITELIST,
+    [
+        '/home/greg/dev/SonataAdminBundle/./src/',
```
Comment (greg0ire) [inline xdebug-filter.php]: Ah wait this feels a bit too personal :P

**Comment by greg0ire** [comment]
Ah no you just have to put `--prepend` before `-c`

**Comment by greg0ire** [comment]
Crashes on 7.1… let's revert to the version that has a check.

**Comment by OskarStark** [comment]
Dev-kit is merged, can we close here @greg0ire ?

**Comment by greg0ire** [comment]
Great!

**Pull request closed without merging**